pkg/services/sqlstore/stats.go

```diff
@@ -2,6 +2,7 @@ package sqlstore
 
 import (
 	"context"
+	"strconv"
```

Comment:
```suggestion
	"fmt"
	"strconv"
	
	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
```

Reply:
:( Augh, again with the "oh, I thought I had pushed that commit". I went for a slightly different route, I think this should work over the different SQLs without needing `migrator`. I'll move this back to being a draft PR.

Reply:
I agree that your approach with `dialect.Quote` is much more preferable